src/ProcessedStory.ts

```diff
@@ -24,8 +22,14 @@ export default class ProcessedStory {
     this.id = rawStory.id;
     this.parameters = {
       skip: normalizeSkip(rawStory.parameters?.axe?.skip, rawStory),
-      disabledRules: normalizeDisabledRules(rawStory.parameters?.axe?.disabledRules, rawStory),
-      waitForSelector: normalizeWaitForSelector(rawStory.parameters?.axe?.waitForSelector, rawStory),
+      disabledRules: normalizeDisabledRules(
+        rawStory.parameters?.axe?.disabledRules,
+        rawStory,
+      ),
+      waitForSelector: normalizeWaitForSelector(
+        rawStory.parameters?.axe?.waitForSelector,
+        rawStory,
+      ),
```

Comment:
Did your editor change all the formatting automatically?

Reply:
should we add a .prettierrc to this repo?

Reply:
Yeah, adding Prettier to this project makes a lot of sense.